src/olympia/discovery/migrations/0007_auto_20200811_1319.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.14 on 2020-08-11 13:19
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('promoted', '0005_auto_20200803_1214'),
+        ('discovery', '0006_auto_20200807_2051'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
I didn't realize that we'd need a migration for this. This doesn't actually alter the db in any way, does it?

Reply:
no, it's just how django keeps track of the models